packages/perseus-core/src/widgets/label-image/label-image-util.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+    PerseusLabelImageMarker,
+    PerseusLabelImageWidgetOptions,
+} from "@khanacademy/perseus-core";
+
+/**
+ * For details on the individual options, see the
+ * PerseusLabelImageWidgetOptions type
+ */
+type LabelImagePublicWidgetOptions = {
+    choices: PerseusLabelImageWidgetOptions["choices"];
+    imageUrl: PerseusLabelImageWidgetOptions["imageUrl"];
+    imageAlt: PerseusLabelImageWidgetOptions["imageAlt"];
+    imageHeight: PerseusLabelImageWidgetOptions["imageHeight"];
+    imageWidth: PerseusLabelImageWidgetOptions["imageWidth"];
+    markers: ReadonlyArray<LabelImageMarkerPublicData>;
+    hideChoicesFromInstructions: boolean;
+    multipleAnswers: boolean;
+    static: boolean;
```

Comment:
Curious why these ones are listed as boolean while the others reference the original widget options. Wouldn't it be good to reference the original type for all of the fields that are unchanged?